app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/presenter/MarvelLandingScreenPresenter.kt

```diff
@@ -0,0 +1,10 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.mvp.presenter
+
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.MarvelLandingScreenContracts
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.view.MarvelLandingScreenView
+
+class MarvelLandingScreenPresenter(val view: MarvelLandingScreenView): MarvelLandingScreenContracts.presenter {
```

Comment:
if you are using contract, then you should instanciate the view interface
`...presenter(val view: MarvelLandingScreenContracts.View)` this should avoid problems then when you mock the view in unit tests

Reply:
done